packages/common/test-builder/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@dxos/test-builder",
+  "version": "0.1.40",
+  "description": "This package defines general purpose test builder ",
```

Comment:
"this package defines..." not necessary :) "Testing framework for ECHO"

Reply:
That is not just for ECHO, it is base TestBuilder and  everybody should use it as base class for their builders